tests/test_mails.py

```diff
@@ -15,4 +17,44 @@ def test_plain_mail():
 def test_html_mail():
     mail = mails.Mail('test', subject='A test email')
     rendered = mail.html(name='World')
-    assert_equal(rendered.strip(), 'Hello <p>World</p>')
\ No newline at end of file
+    assert_equal(rendered.strip(), 'Hello <p>World</p>')
+
+class TestQueuedMail(OsfTestCase):
+    def setUp(self):
+        OsfTestCase.setUp(self)
+        self.user = factories.AuthUserFactory()
+        self.user.is_registered = True
+        self.user.save()
+
+    def test_queue_mail(self):
```

Comment:
This test should be broken down into three individual tests each concisely describing it's purpose e.g. `test_user_no_login_over_four_weeks_gets_email` or some such thing